src/app/tests/suites/certification/Test_TC_CC_3_3.yaml

```diff
@@ -63,14 +63,25 @@ tests:
               - name: "optionsOverride"
                 value: 0
 
+    #delay step is necessary to let the attribute value change over time
+    - label: "Wait 25ms"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 25
+
     - label:
           "Check current hue attribute value matched the value sent by the last
           command"
-      disabled: true
       command: "readAttribute"
       attribute: "current hue"
       response:
-          value: -1
+          constraints:
+              type: uint8
+              minValue: 0
+              maxValue: 254
```

Comment:
But ... this is not checking that the value matches the value sent by the last command!

Reply:
Modified the label as per testplan